server/locale/DE/Pi30/validOptions.js

```diff
@@ -0,0 +1,23 @@
+import { Types } from '../../../types';
+
+export default {
+    text: {
+        logo: {
+            type: [Types.STRING, ['primary', 'alternative', 'inline', 'none']],
+            position: [Types.STRING, ['left', 'right', 'top']]
+        },
+        text: {
+            color: [Types.STRING, ['black', 'white', 'monochrome', 'grayscale|greyscale']],
+            size: [Types.NUMBER, [12, 10, 11, 13, 14, 15, 16]],
+            align: [Types.STRING, ['left', 'right', 'center']],
+            fontFamily: [Types.STRING],
+            fontSource: [Types.ANY]
+        },
+        preset: [Types.STRING, [undefined, 'smallest']]
+    },
+    custom: {
+        markup: [Types.STRING],
+        ratio: [Types.ANY],
+        preset: [Types.STRING, [undefined, 'smallest']]
+    }
```

Comment:
```suggestion
    custom: {
        markup: [Types.STRING],
        ratio: [Types.ANY],
        preset: [Types.STRING, [undefined, 'smallest']]
    }
```
We can remove this `custom` object because this is only in the US. And if you don't mind, can you remove the `custom` object from the DE GPL `validOptions.js` file as well?

Reply:
Also on line 16 right above this we can remove the line `preset: [Types.STRING, [undefined, 'smallest']]` since we no longer use this feature.

Reply:
Would you like the preset array removed from the text object in DE GPL as well?